pkg/processor/runtime/types.go

```diff
@@ -56,6 +57,7 @@ type Configuration struct {
 	TriggerKind              string
 	WorkerTerminationTimeout time.Duration
 	ControlMessageBroker     *controlcommunication.AbstractControlMessageBroker
+	Mode                     functionconfig.TriggerWorkMode
```

Comment:
Why is this needed both here and in the trigger configuration?

Reply:
@TomerShor user configures on trigger level and then we just propagate it to all workers of given trigger